clouddriver-google/src/main/groovy/com/netflix/spinnaker/clouddriver/google/provider/view/GoogleLoadBalancerProvider.groovy

```diff
@@ -52,9 +55,21 @@ class GoogleLoadBalancerProvider implements LoadBalancerProvider<GoogleLoadBalan
     } as Set
   }
 
-  GoogleLoadBalancer.View loadBalancersFromCacheData(CacheData loadBalancerCacheData) {
-    GoogleLoadBalancer loadBalancer = objectMapper.convertValue(loadBalancerCacheData.attributes, GoogleLoadBalancer)
-    GoogleLoadBalancer.View loadBalancerView = loadBalancer?.view
+  GoogleLoadBalancerView loadBalancersFromCacheData(CacheData loadBalancerCacheData) {
+    def loadBalancer = null
+    switch (loadBalancerCacheData.attributes?.type) {
+      case GoogleLoadBalancerType.HTTP.toString():
```

Comment:
This is an object model tradeoff. The only difference in the model so far between HTTP/HTTPS is the certificate field, which is easier to null check against rather than adding extra logic in the caching agent, etc for an additional LB type. It's documented in comments so that assumption is pretty clear.